CelesteNet.Client/Components/CelesteNetChatComponent.cs

```diff
@@ -1,12 +1,16 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Data;
 using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
 using System.Text.RegularExpressions;
 using Celeste.Mod.CelesteNet.Client.Entities;
 using Celeste.Mod.CelesteNet.DataTypes;
 using Microsoft.Xna.Framework;
 using Microsoft.Xna.Framework.Input;
 using Monocle;
+using static Celeste.Mod.CelesteNet.Client.Components.CelesteNetPlayerListComponent;
```

Comment:
That probably shouldn't be here, just to keep everything organized

Reply:
I think that was automatic, oops